src/sentry/templates/sentry/emails/email-styles.html

```diff
@@ -275,6 +275,7 @@
   .inner table th {
     min-width: 60px;
     color: #968ba0;
+    padding: 10px 0 0 0;
```

Comment:
Not sure if this will cause problems other places, but the table headings def feel too high.